backend/pyproject.toml

```diff
@@ -0,0 +1,157 @@
+[tool.isort]
+py_version=312
```

Comment:
here we need to put the python version actually used by the project

Reply:
Hi, I believe it's 3.9, so I just change 312 in 3.9 ?

Reply:
indeed: 312 -> 39

done in https://github.com/XavB64/lowtrip/pull/69/commits/c90d4aff98d74c0067eb04579bda31f5cdc4ee7d